demisto_sdk/commands/validate/validators/base_validator.py

```diff
@@ -225,12 +225,17 @@ def is_error_ignored(
     if err_code not in ignorable_errors:
         return False
     if related_file_type:
-        content_item_as_dict = content_item.dict()
         # If the validation should run on a file related to the main content, will check if the validation's error code is ignored by any of the related file paths.
         for related_file in related_file_type:
-            if related_file_object := content_item_as_dict.get(related_file.value):
-                return err_code in content_item.ignored_errors_related_files(
-                    related_file_object.path
+            try:
+                related_file_object = getattr(content_item, related_file.value)
+                if err_code in content_item.ignored_errors_related_files(
+                    related_file_object.file_path
+                ):
+                    return True
+            except Exception as err:
+                logger.warning(
+                    f"Unable to determine if error code {err_code} should be ignored, got {err}"
```

Comment:
```suggestion
                logger.debug(
                    f"Unable to determine if error code {err_code} should be ignored, got {err}"
```

if this can spam us too, consider changing to `debug` instead, leaving to your call